Gemfile

```diff
@@ -1,4 +1,13 @@
 source 'https://rubygems.org'
 
+if Gem::Version.new(RUBY_VERSION) < Gem::Version.new('2')
+  gem "pry-debugger"
```

Comment:
considering the lock files, I don't think this is working...

Reply:
Not sure what you mean here, this is only the Gemfile for development and not gemspec for distribution. The lockfile shouldn't be included anywhere and gets rebuilt based on the current ruby environment for gem development.

?